content/development/getting-started.html

```diff
@@ -17,7 +20,32 @@ <h3>Getting Involved</h3>
 and preferences are. Usually, by running Haiku for a while you will easily find
 out things that don't work as expected or that can be improved.
 
-<!--more-->
+<h2><a name="development_tools" title="development_tools"></a>What do I need for coding?</h2>
+<ul>
+	<li>
+		Currently, Haiku is the most optimal and frictionless development environment.
+		Linux, BSD, and OS X also work perfectly fine.
```

Comment:
no one says "OS X" anymore and also it isn't this will supported. And "BSD" is in fact only FreeBSD and even then I'm not so sure.

For this "getting started" page I would stick with the simple and short version: either use Linux and a virtual machine, or work from Haiku directly. The other options can be linked in another page but they are not "getting started" level material.

Reply:
The compatibility table in "Building Haiku" says otherwise.